enterprise-suite/es-monitor-api/bootstrap-monitors.json

```diff
@@ -51,7 +51,7 @@
       "model": "threshold",
       "parameters": {
         "name": "node_network_errors",
-        "metric": "node_network_errors",
+        "metric": "node_network_errors_rate",
```

Comment:
this name isn't gramatical, as we don't say "errors rate" but rather we say "error rate"

Please change the name to "node_network_error_rate"

Reply:
I feel like I've been saying this over and over.  Please let me know if there is some kind of disagreement